test/data/D-4449.xcp

```diff
@@ -5,7 +5,7 @@ Handicap="100"
 PolarName="Hornet"
 PolarInformation="80.000,-0.606,120.000,-0.990,160.000,-1.918"
 PolarReferenceMass="318"
-PolarDryMass="302"
+PlaneEmptyMass="302"
```

Comment:
Can you please add another unit test which ensures that reading plane files from old XCSoar versions keeps working? I don't want to break this, this is important.

Reply:
Good hint.